crates/loro-common/src/lib.rs

```diff
@@ -248,18 +248,18 @@ mod container {
 /// instead of moving it to the [`DELETED_TREE_ROOT`].
 ///
 /// This root only can be old parent of node.
-pub const UNEXIST_TREE_ROOT: Option<TreeID> = Some(TreeID {
+pub const UNEXIST_TREE_ROOT: TreeID = TreeID {
```

Comment:
A better way to avoid footgun afterwards may be create an another `TreeIdInner` enum with variances like `Unexist` and `Deleted`. And make it the only way to get and set `TreeID`